### ISSUE TITLE
RHEL 8 page

### ISSUE THREAD

**Issue by alistairmc** [redhat-developer/rhd-frontend#355]
On the RHEL 8 page the curated links are aligned right and look strange, is this correct?

**Comment by gdoyle1** [comment]
Nope, that is not right :)  - I can take a look at the RHEL 8 page we ship today.

FYI @joeschram @sjcox-rh
